server/app/configure/authentication/google.js

```diff
@@ -26,7 +26,8 @@ module.exports = function (app) {
                     return UserModel.create({
                         google: {
                             id: profile.id
-                        }
+                        }, 
+                        email: profile.emails[0].value
```

Comment:
Can we also send the firstname and lastname to our backend User model when we create this user? I *think* Google sends that back.